apps/pwabuilder-vscode/src/services/dashboard/dashboard-view.ts

```diff
@@ -0,0 +1,82 @@
+import * as vscode from "vscode";
+import { getScriptsObject, initDashboard } from "./dev-dashboard";
+
+export class DashboardViewProvider implements vscode.TreeDataProvider<any> {
+    scripts: any = {};
+
+    constructor(private workspaceRoot: string) {
+    }
+
+    getTreeItem(element: DashboardItem): vscode.TreeItem {
+        return element;
+    }
+
+    async getChildren(
+        element?: DashboardItem
+    ): Promise<DashboardItem[] | undefined> {
+        if (!this.workspaceRoot) {
+            vscode.window.showInformationMessage("No dashboard in empty workspace");
```

Comment:
```suggestion
            vscode.window.showInformationMessage("You can't have a dashboard open in an empty workspace.");
```

Complete sentences in warning messages just for clarity.